proto/protocolapis/astria/protocol/transactions/v1alpha1/types.proto

```diff
@@ -246,3 +246,14 @@ message FeeChangeAction {
     astria.primitive.v1.Uint128 ics20_withdrawal_base_fee = 40;
   }
 }
+
+// Response to a transaction fee ABCI query.
+message TransactionFeeResponse {
+  uint64 height = 2;
```

Comment:
Index from 1?

Reply:
we should probably change all of the response types if we want to index from 1, maybe we can do that when we move them to their own package?